test/es-module/test-esm-import-flag.mjs

```diff
@@ -42,6 +43,48 @@ describe('import modules using --import', { concurrency: true }, () => {
     assert.strictEqual(signal, null);
   });
 
+  it('should treat the last import as the main entry point when no argument is passed', async () => {
+    const { code, signal, stderr, stdout } = await spawnPromisified(
+      execPath,
+      [
+        '--import', mjsImport,
```

Comment:
We would want a test with several `--import` and some way to validate that there's a "main" one